[2024-06-24 14:31:29,533] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2023-07-06T00:00:00+00:00 [queued]>
[2024-06-24 14:31:29,571] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2023-07-06T00:00:00+00:00 [queued]>
[2024-06-24 14:31:29,572] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-06-24 14:31:29,574] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-06-24 14:31:29,575] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-06-24 14:31:29,677] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2023-07-06T00:00:00+00:00
[2024-06-24 14:31:29,703] {standard_task_runner.py:52} INFO - Started process 20909 to run task
[2024-06-24 14:31:29,727] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'second_task', '2023-07-06T00:00:00+00:00', '--job-id', '8383', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpve4k6bke', '--error-file', '/tmp/tmpju07nfbg']
[2024-06-24 14:31:29,740] {standard_task_runner.py:77} INFO - Job 8383: Subtask second_task
[2024-06-24 14:31:30,111] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.second_task 2023-07-06T00:00:00+00:00 [running]> on host 799b1e36e3f1
[2024-06-24 14:31:30,359] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2023-07-06T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-06T00:00:00+00:00
[2024-06-24 14:31:30,362] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2024-06-24 14:31:30,368] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2024-06-24 14:31:30,404] {bash.py:169} INFO - Output:
[2024-06-24 14:31:30,406] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2024-06-24 14:31:30,407] {bash.py:177} INFO - Command exited with return code 0
[2024-06-24 14:31:30,680] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20230706T000000, start_date=20240624T143129, end_date=20240624T143130
[2024-06-24 14:31:30,757] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-24 14:31:30,813] {local_task_job.py:146} INFO - Task exited with return code 0
